Fortran/Indirect/setup.py

```diff
@@ -0,0 +1,53 @@
+                                                                  'mantidindirect/Bayes/Four.f90', 'mantidindirect/Bayes/Util.f90'])
+Quest = numpy.distutils.core.Extension(name="Quest",
+                                       sources=['mantidindirect/Bayes/Quest_main.f90',
+                                                'mantidindirect/Bayes/Quest_subs.f90',
+                                                'mantidindirect/Bayes/BlrRes.f90',
+                                                'mantidindirect/Bayes/Bayes.f90',
+                                                'mantidindirect/Bayes/Four.f90',
+                                                'mantidindirect/Bayes/Util.f90',
+                                                'mantidindirect/Bayes/Simopt.f90'])
+QLse = numpy.distutils.core.Extension(name="QLse",
+                                      sources=['mantidindirect/Bayes/QLse_main.f90',
+                                               'mantidindirect/Bayes/QLse_subs.f90',
+                                               'mantidindirect/Bayes/BlrRes.f90',
+                                               'mantidindirect/Bayes/Bayes.f90',
+                                               'mantidindirect/Bayes/Four.f90',
+                                               'mantidindirect/Bayes/Util.f90',
+                                               'mantidindirect/Bayes/Simopt.f90'])
+QLres = numpy.distutils.core.Extension(name="QLres",
+                                       sources=['mantidindirect/Bayes/QLres_main.f90',
+                                                'mantidindirect/Bayes/QLres_subs.f90',
+                                                'mantidindirect/Bayes/BlrRes.f90',
+                                                'mantidindirect/Bayes/Bayes.f90',
+                                                'mantidindirect/Bayes/Four.f90',
+                                                'mantidindirect/Bayes/Util.f90'])
+
+QLdata = numpy.distutils.core.Extension(name="QLdata",
+                                        sources=['mantidindirect/Bayes/QLdata_main.f90',
+                                                 'mantidindirect/Bayes/QLdata_subs.f90',
+                                                 'mantidindirect/Bayes/Bayes.f90',
+                                                 'mantidindirect/Bayes/Four.f90',
+                                                 'mantidindirect/Bayes/Util.f90'])
+muscat = numpy.distutils.core.Extension(name="muscat",
+                                        sources=['mantidindirect/AbsCorrection/muscat_data_main.f90',
+                                                 'mantidindirect/AbsCorrection/muscat_data.f90',
+                                                 'mantidindirect/AbsCorrection/muscat_geom.f90'])
+numpy.distutils.core.setup(
+    name='mantidindirect',
+    ext_modules=[ResNorm, Quest, QLse, QLres, QLdata, muscat],
+    install_requires=['numpy>=1.17.5'],
```

Comment:
Does this require `> 1.17`? RedHat 7 only as 1.13 I think so won't pip try and install a newer numpy? That can be a little dangerous as we compile against numpy...

Reply:
It doesn't strictly require >1.17, but that's what I had built against as its the oldest long term support version at the moment and seemed to be fine for IDaaS. But I didn't realise we were using 1.13 on RedHat 7. I was just having issues building on the docker container with versions less than 1.16 as it was making me build the wheel for numpy as well.

Reply:
Yeah, I can try that. So compiling the wheel for 1.17 and trying to use it with numpy 1.12 didn't work (as Spencer faced this issue, which is when I put the numpy requirements in). But maybe the other way round will be more successful. 

Given that this is an optional dependency I would quite like it to try and install it, but fail if numpy > 1.17 isn't installed (i.e I don't want it to upgrade my existing numpy...I wonder if there's a way to do that.)